tests/integration/test_features.py

```diff
@@ -246,6 +246,16 @@ def test_can_round_trip_binary_custom_content_type(smoke_test_app):
     assert response.content == bin_data
 
 
+def _assert_contains_access_control_allow_methods(headers, methods):
+    actual_methods = headers['Access-Control-Allow-Methods'].split(',')
```

Comment:
Why not just `assert sorted(actual_methods) == sorted(methods)`?  You'd also get a better error message so you could see what the full list is instead of just which one failed.